relay-config/src/config.rs

```diff
@@ -710,6 +706,31 @@ impl Default for Processing {
     }
 }
 
+/// Outcome generation specific configuration values.
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(default)]
+pub struct Outcomes {
+    /// Controls whether outcomes will be emitted when processing is disabled.
+    /// Processing relays always emit outcomes (for backwards compatibility).
+    pub emit_outcomes: bool,
+    /// The maximum number of outcomes that are batched before being sent
+    /// via http to the upstream (only applies to non processing relays)
+    pub max_outcome_batch_size: usize,
+    /// The maximum time interval (in milliseconds) that an outcome may be batched
+    /// via http to the upstream (only applies to non processing relays)
+    pub max_outcome_interval_millsec: u64,
+}
+
+impl Default for Outcomes {
+    fn default() -> Self {
+        Outcomes {
+            emit_outcomes: false,
+            max_outcome_batch_size: 1000,
+            max_outcome_interval_millsec: 500,
```

Comment:
Please shorten these to `batch_size` and `batch_interval`. That makes them equivalent to the keys in `cache.*` and does not repeat the redundant `outcome` from the parent object.

Reply:
Good point, I'll remove the redundant outcome (it was left from the time when these were part of the relay config object).

I would still prefer batch_interval_millsec (or _miilisec ... ). 
I always like to specify the unit of measure when we deal with a number representing something like time so it is clear to the user what he is supposed to put in ( milliseconds, seconds, minutes ... ).  

It would make it inconsistent with other config names but I think this is the correct way to go.

What do you think ?

Reply:
It would make sense, but since all other options do not have this suffix yet I'd personally opt to remove it. I'll leave this up to you. If you keep it, please shorten it to `_ms` though, which should be clear enough.

Reply:
ok, will remove